x/emissions/module/rewards/worker_rewards.go

```diff
@@ -8,8 +8,10 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-var TASK_FORECAST = true
-var TASK_INFERENCE = false
+const (
+	taskForecast  = 1
```

Comment:
not capitalized?

Reply:
Usually we don't use all caps in Go. Do you use all caps here? 

In this particular case, the constants are local, so they start in lowercase letters.